Bottom: f83affe7c238b82b8e9c82f4d5f15b8c379e8c32
Top:    8056b5c93aa56b0526e544bdbe39a58171bb395b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-27 19:19:18 +0000

Refresh of find-c-avoid-modifying-bounds-

---

diff --git a/src/find.c b/src/find.c
index 7f9afd8..be88bab 100644
--- a/src/find.c
+++ b/src/find.c
@@ -764,20 +764,20 @@ LookupLOConnectionsToPVList (bool AndRats)
 
           /* add touching lines */
           if (setjmp (info.env) == 0)
-            r_search (layer->line_tree, &search_box, NULL,
-                      LOCtoPVline_callback, &info);
+            r_search (layer->line_tree, &search_box,
+                      NULL, LOCtoPVline_callback, &info);
           else
             return true;
           /* add touching arcs */
           if (setjmp (info.env) == 0)
-            r_search (layer->arc_tree, &search_box, NULL,
-                      LOCtoPVarc_callback, &info);
+            r_search (layer->arc_tree, &search_box,
+                      NULL, LOCtoPVarc_callback, &info);
           else
             return true;
           /* check all polygons */
           if (setjmp (info.env) == 0)
-            r_search (layer->polygon_tree, &search_box, NULL,
-                      LOCtoPVpoly_callback, &info);
+            r_search (layer->polygon_tree, &search_box,
+                      NULL, LOCtoPVpoly_callback, &info);
           else
             return true;
         }
@@ -785,8 +785,8 @@ LookupLOConnectionsToPVList (bool AndRats)
       if (AndRats)
         {
           if (setjmp (info.env) == 0)
-            r_search (PCB->Data->rat_tree, &search_box, NULL,
-                      LOCtoPVrat_callback, &info);
+            r_search (PCB->Data->rat_tree, &search_box,
+                      NULL, LOCtoPVrat_callback, &info);
           else
             return true;
         }
